frontend/src/ModelOverview.tsx

```diff
@@ -69,21 +71,51 @@ function ModelOverview({ version }: ModelOverviewProps) {
             </Box>
           </Box>
           <Box sx={{ p: 2 }}>
-            <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
+            <Stack spacing={0.5} direction='row'>
+              <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
 
+              <HelpPopover
+                anchorOrigin={{ vertical: 'center', horizontal: 'right' }}
+                transformOrigin={{ vertical: 'center', horizontal: 'left' }}
+              >
+                <Typography sx={{ p: 1 }}>
+                  {schema?.schema.properties.contacts.properties.uploader.description}
+                </Typography>
+              </HelpPopover>
+            </Stack>
             <Typography variant='body1'>
               <EntitiesDisplay entities={version.metadata.contacts.uploader} />
             </Typography>
           </Box>
           <Box sx={{ p: 2 }}>
-            <Typography variant='h6'>{schema?.schema.properties.contacts.properties.reviewer.title}</Typography>
+            <Stack spacing={0.5} direction='row'>
+              <Typography variant='h6'>{schema?.schema.properties.contacts.properties.reviewer.title}</Typography>
 
+              <HelpPopover
```

Comment:
This can also be simplified if you set defaults within the component
```

              <HelpPopover>
                <Typography sx={{ p: 1 }}>
                  {schema?.schema.properties.contacts.properties.reviewer.description}
                </Typography>
              </HelpPopover>
```